src/game.js

```diff
@@ -0,0 +1,60 @@
+function Game() {
+  this.rolls = [];
+  this.score = 0;
+  this.frame = 1;
+  this.frameRollCount = 1;
+}
+
+Game.prototype.roll = function(roll) {
+  if (this.frame >= 10) {
+    this.finalFrame(roll);
+    } else {
+    if (roll === 10){
+      this.rolls.push("X");
+      this.frame++;
+    } else if ((roll + this.rolls[this.rolls.length -1] === 10) && (this.frameRollCount == 2)) {
+      this.rolls.push("/");
+      this.frame++;
+      this.frameRollCount = 1;
+    } else {
+      this.rolls.push(roll);
+      this.frameRollCount === 1 ? this.frameRollCount = 2 : (this.frame++, this.frameRollCount = 1);
+    };
+    this.calculateScore();
```

Comment:
It feels a little hard to read that `calculateScore` is called from different places, depending on whether it's the final frame.  Is there a way to DRY up this call to clarify the code?